[2025-10-16T10:06:03.599+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-16T10:06:03.616+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.dag2_python_1 scheduled__2025-10-16T10:05:00+00:00 [queued]>
[2025-10-16T10:06:03.622+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.dag2_python_1 scheduled__2025-10-16T10:05:00+00:00 [queued]>
[2025-10-16T10:06:03.624+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-16T10:06:03.636+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): dag2_python_1> on 2025-10-16 10:05:00+00:00
[2025-10-16T10:06:03.645+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=238) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-16T10:06:03.648+0000] {standard_task_runner.py:63} INFO - Started process 240 to run task
[2025-10-16T10:06:03.647+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'dag2_python_1', 'scheduled__2025-10-16T10:05:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpi0mkc2ge']
[2025-10-16T10:06:03.650+0000] {standard_task_runner.py:91} INFO - Job 57: Subtask dag2_python_1
[2025-10-16T10:06:03.663+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-16T10:06:03.701+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.dag2_python_1 scheduled__2025-10-16T10:05:00+00:00 [running]> on host bb861ccd12fb
[2025-10-16T10:06:03.786+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='dag2_python_1' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T10:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-16T10:05:00+00:00'
[2025-10-16T10:06:03.788+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-16T10:06:03.789+0000] {logging_mixin.py:188} INFO - bash you have 0 age
[2025-10-16T10:06:03.790+0000] {logging_mixin.py:188} INFO - bash you have 1 age
[2025-10-16T10:06:03.790+0000] {logging_mixin.py:188} INFO - bash you have 2 age
[2025-10-16T10:06:03.791+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-16T10:06:03.792+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-16T10:06:03.801+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=dag2_python_1, run_id=scheduled__2025-10-16T10:05:00+00:00, execution_date=20251016T100500, start_date=20251016T100603, end_date=20251016T100603
[2025-10-16T10:06:03.825+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-16T10:06:03.840+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-16T10:06:03.842+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
